code/languages/C++/Friend_function/README.md

```diff
@@ -0,0 +1,73 @@
+## friend Function
+
+A friend function of a class is defined outside that class' scope but it has the right to access all private and protected members of the class. Even though the prototypes for friend functions appear in the class definition, friends are not member functions.
```

Comment:
defined outside the scope of that class...

Reply:
Done! +1